www_src/components/snackbar/snackbar.less

```diff
@@ -0,0 +1,17 @@
+.snackbar {
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: #444;
+  line-height: 3em;
+  text-align: center;
+  color: #EEE;
+  height: 3em;
```

Comment:
in this case, it kind of has to be, since we need to animate it out of existence. We can, however, use vh or something similar to match what Material Design requires use to use.